install/ansible/uninstall.sh

```diff
@@ -140,6 +140,9 @@ sed -i.bak "s#__CLUSTER_STORE__#$cluster#g" $env_file
 if [ "$aci_image" != "" ]; then
 	sed -i.bak "s#.*aci_gw_image.*#\"aci_gw_image\":\"$aci_image\",#g" "$env_file"
 fi
+if [ "$contiv_v2plugin_install" == "true" ]; then
```

Comment:
How do we set the docker version to ensure that v2plugin is only used with 17.03 or something such? Is it possible for ppl to try v2plugin with an old version of docker?